### ISSUE TITLE
interactive cells should have their own a11y help dialog

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#233920]
cc @jooyoungseo

We now have an a11y help dialog in the repl input. Adding a help dialog like the one for jupyter notebook cells would be a good idea.

**Comment by amunger** [comment]
verification steps:

1. create a python REPL with `start native REPL`
2. execute a cell
3. enable screen reader mode
4. focus on the input box and press alt+f1
5. focus on the prevously executed cell and press alt+f1

the help dialogs should be different depending on the focus in steps 4 and 5
